packages/tanstack-react-router-urql/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@urql/tanstack-react-router",
+  "version": "1.0.0",
+  "description": "Convenience wrappers for using urql with SSR in @tanstack/react-router and @tanstack/start",
+  "sideEffects": false,
+  "homepage": "https://formidable.com/open-source/urql/docs/",
+  "bugs": "https://github.com/urql-graphql/urql/issues",
+  "license": "MIT",
+  "author": "Manuel Schiller",
```

Comment:
This would imply that you are going to publish it, not urql as an author. You might want to refer to other packages for this field

Reply:
done